src/main/java/ee/tuleva/onboarding/capital/InitialCapitalController.java

```diff
@@ -21,8 +21,8 @@
     @ApiOperation(value = "Get info about current user initial capital")
     @RequestMapping(method = GET, value = "/initial-capital")
     @JsonView(InitialCapitalView.SkipUserField.class)
-    public InitialCapital initialCapital(@ApiIgnore @AuthenticationPrincipal User user) {
-        return initialCapitalRepository.findByUser(user);
+    public InitialCapital initialCapital(@ApiIgnore @AuthenticationPrincipal AuthenticatedPerson authenticatedPerson) {
+        return initialCapitalRepository.findByUser(authenticatedPerson.getUser().orElseThrow(RuntimeException::new));
```

Comment:
Create a more specific Exception class that you can throw here instead of using the generic RuntimeException.